<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>689</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>STM Sensorss</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_terminal">
       <attribute name="title">
        <string>Terminal</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="text_browser_0"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_0"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string>Serial</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_26">
             <item>
              <widget class="QPushButton" name="btn_serial_connect">
               <property name="text">
                <string>Connect</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_serial">
               <property name="minimumSize">
                <size>
                 <width>125</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_serial_refresh">
               <property name="text">
                <string>Refresh</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_serial_disconnect">
               <property name="text">
                <string>Disconnect</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string>Terminal</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="radioButton_strings">
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Highlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Highlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="Highlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>240</red>
                   <green>240</green>
                   <blue>240</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Read Strings</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_packages">
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Highlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>242</red>
                   <green>241</green>
                   <blue>240</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Highlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>242</red>
                   <green>241</green>
                   <blue>240</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="Highlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>240</red>
                   <green>240</green>
                   <blue>240</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Read Packages</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_real_time_plots">
       <attribute name="title">
        <string>Real-Time Plots</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QTabWidget" name="tabWidget_plots">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_environment">
           <attribute name="title">
            <string>Environment</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Humidity</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QCustomPlot" name="plot_humidity" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Barometer</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QCustomPlot" name="plot_barometer" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Temperature</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QCustomPlot" name="plot_temperature" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QGroupBox" name="groupBox_6">
              <property name="title">
               <string>Ambient Light</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QCustomPlot" name="plot_ambient_light" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_IMU">
           <attribute name="title">
            <string>IMU</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTabWidget" name="tabWidget_IMU">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_accelerometer">
               <attribute name="title">
                <string>Accelerometer</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QCustomPlot" name="plot_IMU_accelerometer" native="true"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_gyroscope">
               <attribute name="title">
                <string>Gyroscope</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_13">
                <item>
                 <widget class="QCustomPlot" name="plot_IMU_gyroscope" native="true"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_magnetometer">
               <attribute name="title">
                <string>Magnetometer</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QCustomPlot" name="plot_IMU_magnetometer" native="true"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_light">
           <attribute name="title">
            <string>Light</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QGroupBox" name="groupBox_8">
              <property name="title">
               <string>RGB</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QCustomPlot" name="plot_RGB" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_25">
              <property name="title">
               <string>UV</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <widget class="QCustomPlot" name="plot_UV" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_audio">
           <attribute name="title">
            <string>Audio</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_logic_analyser">
       <attribute name="title">
        <string>Logic Analyser</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
      <widget class="QWidget" name="tab_CAN">
       <attribute name="title">
        <string>CAN</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
